packages/terra-core-docs/src/terra-dev-site/doc/button/AnchorButton.doc.mdx

```diff
@@ -0,0 +1,46 @@
+import { Badge } from 'terra-button/package.json?dev-site-package';
```

Comment:
this file name should include numeric value in order to arrange docs in terra-ui site. 
`AnchorButton.3.doc.mdx`

Reply:
Renamed in https://github.com/cerner/terra-core/pull/3655/commits/81401caf927c1b11a30aa72ed0740f8e7e2c6df6